public/locales/en/translation.json

```diff
@@ -133,6 +134,7 @@
 	"ClientFormField": {
 		"Add new input": "Add new input",
 		"Remove input": "Remove input",
+		"Invalid format, should have minimum of 8 characters": "Invalid format, should have minimum of 8 characters",
```

Comment:
@Aringoaway `"Invalid format, input should have minimum of 8 characters":"Invalid format, input should have minimum of 8 characters"`